public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -399,8 +409,13 @@ function queryServiceMapPrometheus(request: DataQueryRequest<PromQuery>, datasou
   );
 }
 
-function serviceMapQuery(request: DataQueryRequest<TempoQuery>, datasourceUid: string) {
-  return queryServiceMapPrometheus(makePromServiceMapRequest(request), datasourceUid).pipe(
+function firstServiceMapQuery(request: DataQueryRequest<TempoQuery>, datasourceUid: string) {
+  const serviceMapRequest = makePromServiceMapRequest(request);
+  serviceMapRequest.targets = makeApmRequest([buildExpr(rateMetric, '', request)]).concat(
```

Comment:
We should only run these queries and display the table in the UI if the `tempoApmTable` toggle is enabled

Nit: Function names could be more descriptive (`firstServiceMapQuery` and `secondServiceMapQuery`)